libs/damap/src/lib/damap.module.ts

```diff
@@ -9,11 +9,15 @@ import { TranslateModule } from '@ngx-translate/core';
 import { DamapStoreModule } from './store/damap-store.module';
 import { APP_ENV } from './constants';
 import { GdprComponent } from './components/gdpr/gdpr.component';
+import { DamapInfoComponent } from './components/damap-info/damap-info.component';
+import { DmpInstructionsComponent } from './components/dmp-instructions/dmp-instructions.component';
 
 export const DAMAP_ROUTES: Route[] = [
   { path: '', component: DashboardComponent },
   { path: 'dashboard', component: DashboardComponent },
   { path: 'plans', component: PlansComponent },
+  { path: 'damap', component: DamapInfoComponent },
+  { path: 'how-to-create', component: DmpInstructionsComponent },
```

Comment:
Would it make sense to group these sites under a `info` prefix?

Reply:
Yes, I will add the prefix to them